source/tests/pd/model/test_dpa1.py

```
@@ -0,0 +1,164 @@
+                "float64",
+            ],  # precision
+            [False, True],  # smooth_type_embedding
+            [
+                False,
+            ],  # type_one_side
+            ["concat", "strip"],  # tebd_input_mode
+            [False, True],  # use_econf_tebd
+        ):
+            dtype = PRECISION_DICT[prec]
+            rtol, atol = get_tols(prec)
+            err_msg = f"idt={idt} prec={prec}"
```

Ruff (0.8.0)

Local variable `err_msg` is assigned to but never used

Remove assignment to unused variable `err_msg`

(F841)